src/components/SamlConfiguration/index.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Index from './index';
+import LmsApiService from './../../data/services/LmsApiService';
+import { REQUIRED_DATA_FIELDS } from './SamlProviderDataForm';
+
+const formData = new FormData();
+REQUIRED_DATA_FIELDS.forEach(field => formData.append(field, 'testdata'));
```

Comment:
Can I suggest doing
```
const responseData = { data: {} };
const oldConfig = { };
REQUIRED_DATA_FIELDS.forEach((field) => {
  formData.append(field, 'testdata');
  responseData.data[field] = 'testdata');
  oldConfig[field] = 'oldtestdata');
})
```
so that you don't have to loop through formData 3+ more times?